py3status/module.py

```diff
@@ -311,14 +311,34 @@ def set_module_options(self, module):
         self.py3_module_options = {}
         mod_config = self.config["py3_config"].get(module, {})
 
+        if "min_length" in mod_config:
+            min_length = mod_config["min_length"]
+            if not isinstance(min_length, int):
+                err = 'invalid "min_length" attribute should be an int'
+                raise TypeError(err)
+
+            self.py3_module_options["min_length"] = min_length
+            self.random_int = randint(0, 1)
+
+            if "position" in mod_config:
+                position = mod_config["position"]
+                if not (
+                    isinstance(position, basestring)
+                    and position.lower() in ("left", "center", "right")
+                ):
+                    err = 'invalid "position" attribute, valid values are:'
+                    err += " left, center, right"
```

Comment:
imho `("left", "center", "right")` should be a constant and used here and above so they are always in sync

Reply:
same, in constants.py plz